site/content/extend/plugins/webapp/_index.md

```diff
@@ -5,4 +5,48 @@ subsection: Plugins
 weight: 20
 ---
 
+# Web App Plugins
+
 Web app plugins are capable of overriding and extending React UI components of the Mattermost web and desktop apps.
+
+Looking for a quick start? [See our "Hello, world!" guide](/extend/plugins/webapp/hello-world/).
+
+Want the web app plugin reference doc? [Find it here](/extend/plugins/webapp/reference/).
+
+## Features
+
+#### Override React Components
+
+Use your own implementations of web app React components to replace the default Mattermost implementations.
+
+For example, if you want to build a custom implementation of the profile popover (sometimes called the profile hovercard) that appears when a user's profile picture is clicked on, simply write your own implementation of the [ProfilePopover](/extend/plugins/webapp/reference/#profilepopover) component and include it in your plugin.
+
+For an example, see the [hovercardexample plugin](https://github.com/jwilander/hovercardexample).
```

Comment:
Maybe we should move this to the mattermost org.

Reply:
Yeah, I'll ask CS to move it over and update the link when it's done